com.github.iwalton3.jellyfin-mpv-shim.json

```diff
@@ -0,0 +1,250 @@
+{
+  "app-id": "com.github.iwalton3.jellyfin-mpv-shim",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.36",
+  "sdk": "org.gnome.Sdk",
+  "command": "jellyfin-mpv-desktop",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=all",
+    "--share=network",
+    "--socket=pulseaudio",
+    "--filesystem=xdg-pictures",
+    "--talk-name=org.gtk.vfs",
+    "--talk-name=org.gtk.vfs.*",
+    "--talk-name=org.gnome.SettingsDaemon.MediaKeys",
+    "--env=LC_NUMERIC=C",
+    "--persist=.config/jellyfin-mpv-shim"

```

Comment:
This is not needed. Your application should respect $XDG_CONFIG_HOME and put config files there.

Reply:
The persist rule has been removed.